ydb/docs/en/core/concepts/backup-collections.md

```diff
@@ -0,0 +1,308 @@
+    │   └── table_3/
+    └── 20250821080000Z_incremental/ # Second incremental
+        └── table_3/
+```
+
+Each backup contains:
+
+- Table schemas at backup time.
+- Data files (full or incremental changes).
+
+### Storage backends {#storage-backends}
+
+#### Cluster storage {#cluster-storage}
+
+### Internal storage
+
+Backups are stored within the {{ ydb-short-name }} cluster itself, providing:
+
+- **High availability**: Leverages cluster replication and fault tolerance.
+- **Performance**: Fast backup and restore operations.
+- **Integration**: Seamless integration with cluster operations.
+- **Security**: Uses cluster security mechanisms.
+
+```sql
+WITH ( STORAGE = 'cluster' )
+```
+
+#### External storage {#external-storage}
+
+Currently, external storage requires manual export/import operations. Use [export/import operations](../reference/ydb-cli/export-import/index.md) to move backups to external storage systems.
+
+Example commands to export a backup collection:
+
+```bash
+# Export entire collection directory to S3
+ydb export s3 --s3-endpoint storage.yandexcloud.net \
+  --bucket my-backups \
+  --item src=.backups/collections/my_collection,dst=my_collection
+
+# Or export to filesystem
+ydb tools dump --path .backups/collections/my_collection \
+  --output /path/to/local/backup
+```
+
+### Background operations {#background-operations}
+
+All backup operations run asynchronously in the background, allowing you to:
+
+- Continue normal database operations during backups.
+- Monitor progress using {{ ydb-short-name }} CLI operation commands.
+- Handle large datasets without blocking other activities.
+
+## How backup collections work internally {#how-they-work}
+
+### Backup creation process {#backup-creation-process}
+
+1. **Transaction isolation**: Backup starts from a consistent snapshot point
+2. **Change tracking**: For incremental backups, only changes since last backup are captured and stored in a [changefeed](cdc.md)
+3. **Change materialization**: When incremental backup is called, the changefeed is compacted into incremental backup tables
+
+### Incremental backup mechanism {#incremental-backup-mechanism}
+
+{% note info %}
+
+Incremental backups use change tracking to identify:
+
+- **New rows**: Added since last backup.
+- **Modified rows**: Changed data in existing rows.  
+- **Deleted rows**: Removed data (tombstone records).
+- **Schema changes**: Currently not supported.
+
+{% endnote %}
+
+### Relation to incremental backups {#relation-to-incremental-backups}
+
+Backup collections are the foundation for incremental backup functionality:
+
+- **Collections enable incrementals**: You must have a collection to create incremental backups.
+- **Chain management**: Collections manage the sequence of full and incremental backups.
+- **Consistency**: All tables in a collection are backed up consistently.
+
+Without backup collections, only full export/import operations are available.
```

Comment:
This doesn't seem to add any new information beyond what has been covered above.